File: platform/services/account/tests/config/env.py
# Copyright (C) 2022-2025 Intel Corporation
# LIMITED EDGE SOFTWARE DISTRIBUTION LICENSE

import os

account_service_grpc_gateway_address = os.getenv("ACCOUNT_SERVICE_GRPC_GATEWAY_ADDRESS", "127.0.0.1:5002")
s3_public_address = os.getenv("S3_PUBLIC_ADDRESS", "127.0.0.1:8333")
# noinspection HttpUrlsUsage
protocol = "http://"
api_v1_prefix = "/api/v1"

FF_MANAGE_USERS = os.getenv("FEATURE_FLAG_MANAGE_USERS", False)
FF_MANAGE_USERS_ROLES = os.getenv("FEATURE_FLAG_MANAGE_USERS_ROLES", False)
FEATURE_FLAG_REQ_ACCESS = os.getenv("FEATURE_FLAG_REQ_ACCESS", False)

File: interactive_ai/libs/iai_core_go/env/env.go
// Copyright (C) 2022-2025 Intel Corporation
// LIMITED EDGE SOFTWARE DISTRIBUTION LICENSE

package env

import (
	"os"
)

// GetEnv tries to get an environment variable. If it is not found the fallback value is returned.
func GetEnv(key string, fallback string) string {
	if value, ok := os.LookupEnv(key); ok {
		return value
	}
	return fallback
}
